interbtc-spec/docs/source/spec/issue.rst

```diff
@@ -215,14 +214,18 @@ Specification
         * If the Vault IS NOT liquidated and has sufficient collateral:
 
             * The Vault's ``toBeIssuedTokens`` MUST increase by the surplus (``amountTransferred - issue.amount``).
+            * The ``issue.fee`` MUST be updated to the amount transferred multiplied by the :ref:`issueFee`.
+            * The ``issue.amount`` MUST be set to the amount transferred minus the updated ``issue.fee``.
 
         * If the Vault IS NOT liquidated and does not have sufficient collateral:
 
             * There MUST exist a :ref:`refund-protocol` request which references ``issueId``.
 
-* The Vault's ``toBeIssuedTokens`` MUST decrease by the total amount.
-* The Vault's ``issuedTokens`` MUST increase by the total amount.
-* The user MUST receive interBTC in it's free balance.
+* The ``issue.status`` MUST be set to ``Completed``.
+* The Vault's ``toBeIssuedTokens`` MUST decrease by ``issue.amount + issue.fee``.
+* The Vault's ``issuedTokens`` MUST increase by ``issue.amount + issue.fee``.
+* The user MUST receive ``issue.amount`` interBTC in its free balance.
+* The Vault MUST receive ``issue.fee`` interBTC in its free balance.
```

Comment:
This is not correct, it is sent to the rewards pool.

Reply:
What's the best way to specify this? `The distributeRewards function MUST be called with issue.fee`, or `the fee pool MUST increase by issue.fee`?

Reply:
I think the latter for now.